HW_6/Task_2_3.py

```diff
@@ -0,0 +1,122 @@
+# Добавьте в пакет, созданный на семинаре шахматный модуль. Внутри него напишите код, решающий задачу о 8 ферзях.
+# Известно, что на доске 8×8 можно расставить 8 ферзей так,
+# чтобы они не били друг друга. Вам дана расстановка 8 ферзей на доске,
+# определите, есть ли среди них пара бьющих друг друга.
+# Программа получает на вход восемь пар чисел, каждое число от 1 до 8 - координаты 8 ферзей.
+# Если ферзи не бьют друг друга верните истину, а если бьют - ложь.
+import random
+
+def make_desk(size: int) -> list[[int]]:
+    EMPTY = 0
+    return [[EMPTY for _ in range(size)] for _ in range(size)]
+
+
+def show_desk(desk: list[[int]]) -> None:
+    print()
+    print(*desk, sep='\n')
+
+
+def feel_desk(desk: list[[int]], queens: list(())) -> list[[int]]:
+    '''
+    Функция заполняет доску ферзями.
+    1 - ферзь на поле, 0 - поле пустое
+    Возвращает доску в виде списка с расставленными фигурами
+    '''
+    BUSY = 1
+    for i in range(len(queens)):
+        desk[queens[i][0]][queens[i][1]] = BUSY
+    return desk
+
+
+def _check_turns(desk: list[[int]]) -> bool:
+    counter = 0
+    for col in desk:
+        for el in col:
+            if el == 1:
+                counter += 1
+        if counter > 1:
+            return False
+        else:
+            counter = 0
+    return True
+
+def _transparent(desk: list[[int]]) -> bool:
+    return [[*col] for col in zip(*desk)]

```

Comment:
Если используете аннотацию типов, то следуйте ей 
```suggestion
def _transparent(desk: list[[int]]) -> list:
    return [[*col] for col in zip(*desk)]

```